sklearn/utils/sparsefuncs.py

```diff
@@ -25,6 +26,49 @@ def _raise_error_wrong_axis(axis):
         raise ValueError(
             "Unknown axis value: %d. Use 0 for rows, or 1 for columns" % axis)
 
+def csr_csc_min_axis0(X):
```

Comment:
Could this be implemented as a backport somehow then?

Also I think you can reduce the amount of code here, because the min of a csc axis 0 is the same as the min of a csr axis 1, and viceversa.

(The scipy implementation uses generic reducers and ufuncs, which might be overkill to backport. But we need to make sure that this is treated as a temporary solution that will be removed when we drop the old scipy.)